.docker/install-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+sudo apt-get -y install -y libxpm4 libxrender1 libgtk2.0-0 libnss3 libgconf-2-4 chromium xvfb gtk2-engines-pixbuf xfonts-cyrillic xfonts-100dpi xfonts-75dpi xfonts-base xfonts-scalable imagemagick x11-apps
```

Comment:
We'd like to use Dusk, but never got round to making it work; chromium xvfb etc may be redundant in this build. We can keep 'em in for when we do need it, maybe?